test/sagemaker_tests/huggingface/inference/integration/sagemaker/test_sm_trained_model.py

```diff
@@ -30,15 +31,23 @@
 @pytest.mark.cpu_test
 def test_sm_trained_model_cpu(sagemaker_session, framework_version, ecr_image, instance_type):
     instance_type = instance_type or "ml.m5.xlarge"
-    _test_sm_trained_model(sagemaker_session, framework_version, ecr_image, instance_type, model_dir)
+    try:
+        _test_sm_trained_model(sagemaker_session, framework_version, ecr_image, instance_type, model_dir)
+    except UnexpectedStatusException as e:
+        dump_logs_from_cloudwatch(e)
+        raise
 
 
 @pytest.mark.model("tiny-distilbert")
 @pytest.mark.processor("gpu")
 @pytest.mark.gpu_test
 def test_sm_trained_model_gpu(sagemaker_session, framework_version, ecr_image, instance_type):
     instance_type = instance_type or "ml.p2.xlarge"
-    _test_sm_trained_model(sagemaker_session, framework_version, ecr_image, instance_type, model_dir)
+    try:
+        _test_sm_trained_model(sagemaker_session, framework_version, ecr_image, instance_type, model_dir)
+    except UnexpectedStatusException as e:
+        dump_logs_from_cloudwatch(e)
```

Comment:
nit: This is a nit because the `dump_logs` function doesn't affect the outcome of the test. However, I would recommend passing the region of the `sagemaker_session`/`ecr_image`, which should be available through the `region` conftest parameter, to the `dump_logs` function so that we can use it with SM Inference tests in all regions.